### DISCUSSION TITLE
Mongo support

### DISCUSSION THREAD

**Discussion by ChristPetitjean** [microcks/microcks#804] [Ideas]
Hello,

A very simple idea.

What do you think of virtualizing some mongo calls to ensure micro-services compliance without mongo deps ? It can avoid dealing with specific infra and is actually not performed but other tools. It can be a key in choosing Microcks over others !

Feel free to give your opinion

**Comment by lbroudoux** [comment]
Hi @ChristPetitjean 

That's a super good idea and we started discussing it a bit in the context of enhancing developer experience, especially for a [Quarkus Dev Service](https://github.com/orgs/microcks/projects/1/views/1?pane=issue&itemId=26478003).

As of today, we already use "fake MongoDB" for our unit tests - to be precise we're using `de.bwaldvogel:mongo-java-server` - but didn't get through a comprehensive list of options. Do you have some experience and recommendations on the different solutions?

Also it could be nice to identity the use-cases you have in mind where less dependency would make Microcks the killer app 😉  Looking forward to pursuing the discussion!
